core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -653,6 +653,19 @@ class ZkUtils(val zkClient: ZkClient,
     (dataStr, stat)
   }
 
+  def readData(path: String, returnNoneIfPathNotExists: Boolean): Option[String] = {
```

Comment:
How about use this method in this PR `zkUtils.readDataMaybeNull(entityConfigPath)._1` and then create a new ticket and refactor the already existing readData method to return only the data without the stat and all the other methods mentioned above. 
The existing readData method https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/ZkUtils.scala#L650 now throws exception, and I found one place where we are using this feature https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/ZkUtils.scala#L650